widgets/BigQueryTable-batchsink.json

```diff
@@ -70,6 +70,25 @@
             ]
           }
         },
+        {
+          "widget-type": "radio-group",
```

Comment:
for booleans, there is a 'toggle' widget that we can use

Reply:
Fixed!